base_name_search_improved/models/ir_model.py

```diff
@@ -18,6 +18,7 @@ def _get_rec_names(self):
 
 
 def _extend_name_results(self, domain, results, limit):
+    limit = 100 if limit is None else limit
```

Comment:
I don't think we need to specifically compare to None, so the more Pythonic idiom can be used
```suggestion
    limit = limit or 100
```

Reply:
You are right thanks :)